pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -581,9 +581,9 @@ type MonZoneSpec struct {
 
 // MgrSpec represents options to configure a ceph mgr
 type MgrSpec struct {
-	// Count is the number of manager to run
+	// Count is the number of manager daemons to run
 	// +kubebuilder:validation:Minimum=0
-	// +kubebuilder:validation:Maximum=2
+	// +kubebuilder:validation:Maximum=5
```

Comment:
if we have a validation by CRD, why do we need secendory validation in the code?

Reply:
Now the maxMgrCount is removed from code and am querying for the existing mgrs to see if there are any extra to remove.